modules/health_quest/app/controllers/health_quest/v0/pgd_questionnaires_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module HealthQuest
+  module V0
+    class PgdQuestionnairesController < HealthQuest::V0::BaseController
+      def show
+        head :ok
```

Comment:
Is this just a placeholder status for the time being? Will it return something else in the future?

Reply:
@LindseySaari yes, it's a placeholder for now. It will be returning questionnaires data in the nearby future.

Reply:
Ok great, are there any api docs for this endpoint or will that be added when the questionnaires data gets implemented?

Reply:
@LindseySaari API docs(Swagger) will be added separately. I will update the description to reflect that. Thanks!

Reply:
Great, thank you!